### PR TITLE
zh trans preemption.md

### PR DESCRIPTION
**Pull request by congfairy**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/cn/docs/reference/glossary/preemption.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Preemption
+id: preemption
+date: 2019-01-31
+full_link: /docs/concepts/configuration/pod-priority-preemption/#preemption
+short_description: >
+  Preemption logic in Kubernetes helps a pending Pod to find a suitable Node by evicting low priority Pods existing on that Node.
```
Comment (xiangpengzhao) [inline content/cn/docs/reference/glossary/preemption.md]: 这里的这句话也翻译一下吧。

File: content/cn/docs/reference/glossary/preemption.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+aka:
+tags:
+- operation
+---
+<!--
+ Preemption logic in Kubernetes helps a pending Pod to find a suitable Node by evicting low priority Pods existing on that Node.
+-->
+
+Kubernetes中的抢占逻辑通过驱逐该节点上存在的低优先级Pod来帮助待处理的Pod找到合适的节点。
```
Comment (xiangpengzhao) [inline content/cn/docs/reference/glossary/preemption.md]: 中英文之间的空格。

File: content/cn/docs/reference/glossary/preemption.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+<!--more-->
+
+<!--
+If a Pod cannot be scheduled, the scheduler tries to [preempt](/docs/concepts/configuration/pod-priority-preemption/#preemption) lower priority Pods to make scheduling of the pending Pod possible.
+-->
+
+
+如果无法调度 Pod，则调度程序会尝试 [preempt](/docs/concepts/configuration/pod-priority-preemption/#preemption) 优先级较低的 Pod ，以便可以调度挂起的 Pod 。
```
Comment (xiangpengzhao) [inline content/cn/docs/reference/glossary/preemption.md]: `preempt` 翻译成中文是不是更好点？
这里的 `pending` 和上文的 `pending` 翻译是不是统一起来？

**Comment by xichengliudui** [comment]
Can you update this pr? @congfairy

File: content/cn/docs/reference/glossary/preemption.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Kubernetes 中的抢占逻辑通过驱逐该节点上存在的低优先级 Pod 来帮助挂起的 Pod 找到合适的节点。
+
+<!--more-->
+
+<!--
+If a Pod cannot be scheduled, the scheduler tries to [preempt](/docs/concepts/configuration/pod-priority-preemption/#preemption) lower priority Pods to make scheduling of the pending Pod possible.
+-->
+
+
+如果无法调度 Pod，则调度程序会尝试 [抢占](/docs/concepts/configuration/pod-priority-preemption/#preemption) 优先级较低的 Pod ，以便可以调度挂起的 Pod 。
```
Comment (xichengliudui) [inline content/cn/docs/reference/glossary/preemption.md]:
```suggestion
如果无法调度 Pod，则调度程序会尝试[抢占](/docs/concepts/configuration/pod-priority-preemption/#preemption) 优先级较低的 Pod，以便可以调度挂起的 Pod。
```

**Comment by tanjunchen** [comment]
@congfairy   
image

**Pull request closed without merging**